packages/remix-architect/package.json

```diff
@@ -17,6 +17,10 @@
     "@remix-run/node": "1.3.2",
     "@remix-run/server-runtime": "1.3.2"
   },
+  "peerDependencies": {
+    "react": ">=16.8",
+    "react-dom": ">=16.8"
+  },
```

Comment:
None of our adapters (`remix-architect`, `remix-cloudflare-pages`, `remix-cloudflare-workers`, `remix-express`, `remix-netlify`) depend on `react` nor `react-dom`. Is it intentional do add them as `peerDependencies` to all of these packages?

Reply:
Yarn complained about those, because react and react-dom are indirect peer dependencies of all of these packages, which means that those dependencies should be declared as peer dependencies of your package as well.
E.g. for `remix-architect` via `remix-server-runtime` → `remix-router`, which has a peer dependency on `react >= 16.8`.

Reply:
Currently these are indeed in `peerDependencies` of `react-router`, which is in `dependencies` of `@remix-run/server-runtime`.

However, we're not using any `react`-related things from `react-router`, so having it in `peerDependencies` isn't necessary.

In the (near) future, we'll have `@remix-run/router`, which will be a framework agnostic version of `react-router`.
Whenever we have this, we'll update `@remix-run/server-runtime`'s `dependencies` to include the new package.

We can remove them from all other packages for now.